src/components/instantlaunches/launch/index.js

```diff
@@ -61,20 +68,34 @@ const InstantLaunchStandalone = (props) => {
         },
     });
 
-    const isLoading = isQueryLoading([status, isFetchingUsageSummary]);
+    const { isFetching: isLoadingResource, error: resourceError } =
+        useDataDetails({
+            paths: [resource_path],
+            enabled: !!resource_path,
+            onSuccess: (resp) => {
+                setResource(resp?.paths[resource_path]);
+            },
+        });
+
+    const isLoading = isQueryLoading([
+        status,
+        isLoadingResource,
+        isFetchingUsageSummary,
+    ]);
 
     if (isLoading) {
         return <LoadingAnimation />;
-    } else if (error) {
+    } else if (error || resourceError) {
+        const err = error || resourceError;
         return (
             <>
                 <ErrorTypography
-                    errorMessage={error.message}
+                    errorMessage={err.message}
                     onDetailsClick={() => setErrorDialogOpen(true)}
                 />
                 <DEErrorDialog
                     open={errorDialogOpen}
-                    errorObject={error}
+                    errorObject={err}
                     handleClose={() => setErrorDialogOpen(false)}
                 />
             </>
```

Comment:
I would replace this with an `ErrorTypographyWithDialog`. Also, the pattern we've been using with the error dialogs is to use a more general error message in the pop up, such as `t("instantLaunchError")`, and then the user can click the error dialog button to view more details to see if it was due to fetching resource info or trying to run the instant launch.

Reply:
Yeah, that's probably good. I admittedly saw the `errorHandler` prop to that and didn't want to figure that out just then, so did this, but given that's passed through to `DEErrorDialog` anyway now that I look, no reason to avoid it. I'll change to the wrapper & the more generic message (whether that one or a new one).